.github/workflows/README.md

```diff
@@ -132,7 +134,7 @@ This is done to prevent pro needing to be published prior to CI runs in budiabse
   - backend-core lives in the monorepo, so it can't be released independently to be used in pro
   - therefore the only option is to pull pro from source and release it as a part of the monorepo release, as if it were a mono package
 
-The install is performed using the same steps as local development, via the `yarn bootstrap` command, see the [Contributing Guide#Pro](../CONTRIBUTING.md#pro)
+The install is performed using the same steps as local development, via the `yarn clean && yarn bootstrap` command, see the [Contributing Guide#Pro](../../docs/CONTRIBUTING.md#pro)
```

Comment:
`yarn bootstrap` alone will perform the needed dependency linking, having the `clean` here might lead someone to believe that it is required

Reply:
ok! I thought it was necessary because when I built the pro version I had to do it. But maybe it was just an isolated case. Removed the "clean" command. thanks!